XenAdmin/Alerts/Types/XenCenterUpdateAlert.cs

```diff
@@ -102,15 +102,20 @@ public override string HelpID
         }
 
         public override void Dismiss()
-        {            
-            Properties.Settings.Default.LatestXenCenterSeen = NewVersion.VersionAndLang;
+        {
+            List<string> current = new List<string>(Properties.Settings.Default.LatestXenCenterSeen.Split(','));
+            if (current.Contains(NewVersion.VersionAndLang))
+                return;
+            current.Add(NewVersion.VersionAndLang);

```

Comment:
This list is now going to grow forever, I suggest we limit the max length. (As the previously dismissed ones are not really needed to be kept.)